front/lib/api/assistant/jit_actions.ts

```diff
@@ -55,6 +58,58 @@ export async function isJITActionsEnabled(
   return use;
 }
 
+export function isConversationQueryableFileContentType(
+  contentType: SupportedContentFragmentType
+): boolean {
+  if (isSupportedImageContentType(contentType)) {
+    return false;
+  }
+  // For now we only allow including text files.
+  switch (contentType) {
+    case "application/msword":
+    case "application/vnd.openxmlformats-officedocument.wordprocessingml.document":
+    case "application/pdf":
+    case "text/markdown":
+    case "text/plain":
+    case "dust-application/slack":
+    case "text/tab-separated-values":
+    case "text/tsv":
+      return false;
+
+    case "text/comma-separated-values":
+    case "text/csv":
+      return true;
+    default:
+      assertNever(contentType);
+  }
+}
+
+export function isConversationSearchableFileContentType(
+  contentType: SupportedContentFragmentType
+): boolean {
+  if (isSupportedImageContentType(contentType)) {
+    return false;
+  }
+  // For now we only allow including text files.
+  switch (contentType) {
+    case "application/msword":
+    case "application/vnd.openxmlformats-officedocument.wordprocessingml.document":
+    case "application/pdf":
+    case "text/markdown":
+    case "text/plain":
+    case "dust-application/slack":
+    case "text/tab-separated-values":
```

Comment:
the next 2 should be false

Reply:
tsv is handled as a doc (flip side of your other question)

Why not slack thread ?